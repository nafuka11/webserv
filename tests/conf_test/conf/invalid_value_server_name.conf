server {
    listen 4242;
    server_name second_server_should_have_server_name;
}

server {
    listen 4242;
}
