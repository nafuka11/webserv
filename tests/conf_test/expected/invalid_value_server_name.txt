webserv: invalid value in "server_name" directive
